resources/service.rb

```diff
@@ -0,0 +1,45 @@
+# Copyright:: Copyright (c) 2017 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+resource_name :hab_service
+provides :hab_service
+
+property :service_name, String, name_property: true
+property :sup_options, [Array, String]
+
+action :load do
+  hab('service', 'load', service_name, sup_options)
```

Comment:
Two things:
1) `hab service` is really just an alias to `hab sup` -- we probably should just use `hab sup` since most of the subcommands are documented in that command.
2) We may need to add some sanity checking here until some of the bugs around loading and unloading are resolved. The biggest issue in 0.20.0 at the moment is if `load` or `unload` is run against a service more than once, hab will throw an error and return a non-zero status after the first execution. Since hab also does not currently have a `status` command to validate this idempotently this will likely be a problem for users when they do something like:

```ruby
hab_service 'origin/servicename' do
  action [:load, :start]
end
```
What we can do in the meantime, is at least validate the existence or non-existence of `/hab/sup/#{new_resource.service_group}/specs/#{new_resource.service_name}.spec`. Note, we will likely have to extract the service group as a separate key/value option from `sup_options` unless you want to try and map and fetch it from the array.